workspace/TS100/translations

```diff
@@ -0,0 +1 @@
+../../Translation Editor
```

Comment:
Is this supported cleanly on all 3 major operating systems?

Would prefer to maintain support for windows

Reply:
What do you think about renaming the directory "Translation Editor" to just "Translation(s)".
This would more ergonomic in general (IMO) and work around this issue, too.

Reply:
Would prefer a folder rename :)